spokestack/asr/cloud_client.py

```diff
@@ -140,14 +137,17 @@ def initialize(self) -> None:
 
     def disconnect(self) -> None:
         """ disconnects client socket connection """
-        if self._socket:
-            self.close()
         self._socket = None
```

Comment:
for an explicit disconnect, you still want to call `socket.close` instead of waiting for the garbage collector to do it